app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitListFragment.java

```diff
@@ -219,9 +219,10 @@ protected void openAddDialogBox() {
 
     // note: add this to list fragment class once swipe is complete in habit event list
     protected void openEditDialogBox(int position) {
-        EditHabitFragment editHabitFragment = new EditHabitFragment(position,
-                _habitItemAdapter._snapshots.get(position),
-                this);
+        EditHabitDialog editHabitFragment = new EditHabitDialog(position,
+                _habitItemAdapter._snapshots.get(position));
+        editHabitFragment.setCancelable(true);
+        editHabitFragment.setTargetFragment(this, 1);
```

Comment:
Changed way of how EditHabitDialog refers to parent fragment to match AddHabitDialog